### ISSUE TITLE
Is it necessary to make:auth?

### ISSUE THREAD

**Issue by haoa2** [Hesto/multi-auth#110]
It's not described on the README.md if it is necessary to run the `php artisan make:auth` command in order to use this package. Just asking because the trailing migrations that are left by that command. Is it necessary or not? Should I remove those migrations and everything will be ok? 

Thanks in advance!

**Comment by haoa2** [comment]
Is it worth mentioning it on the readme?

**Comment by KimKJDesigns** [comment]
Not really, you can use the make:auth command, if you want. But there wouldn't be any continuity in the two auth types. You figure it out pretty quickly that you'd use jus the multi-auth, since it should be used at the start of the project.

**Comment by shiroamada** [comment]
Just be alerted, if you using sub-domain, make sure your `Auth::routes();` is after the subdomain code. If not Laravel will use the default login route.

**Comment by prabhat2k15** [comment]
It works perfectly fine without make:auth. However i have got one issue but it is resolved easily after googleing it :)
